# Generated by Django 3.1.1 on 2020-09-10 08:26

import datetime
import django.contrib.gis.db.models.fields
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('crust', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='store',
            name='coords',
            field=django.contrib.gis.db.models.fields.PointField(default='', srid=4326),
        ),
        migrations.AddField(
            model_name='suburb',
            name='coords',
            field=django.contrib.gis.db.models.fields.PointField(default='', srid=4326),
        ),
        migrations.AlterField(
            model_name='admin',
            name='created_at',
            field=models.DateTimeField(default=datetime.datetime.now),
        ),
        migrations.AlterField(
            model_name='comment',
            name='commented_at',
            field=models.DateTimeField(default=datetime.datetime.now),
        ),
        migrations.AlterField(
            model_name='post',
            name='posted_at',
            field=models.DateTimeField(default=datetime.datetime.now),
        ),
        migrations.AlterField(
            model_name='reply',
            name='replied_at',
            field=models.DateTimeField(default=datetime.datetime.now),
        ),
        migrations.AlterField(
            model_name='systemerror',
            name='occurred_at',
            field=models.DateTimeField(default=datetime.datetime.now),
        ),
    ]
